src/pages/base/wallets/Wallet.js

```diff
@@ -1,24 +1,30 @@
 import classnames from "classnames";
 import makeBlockie from "ethereum-blockies-base64";
-import React from "react";
+import React, { useState } from "react";
+import { Col, Row } from "reactstrap";
 import { useLocation, useParams } from "react-router-dom";
 import Address from "../../../components/Address/Address.js";
 import Loader from "../../../components/Loader/Loader.js";
+
 import { withErrorBoundary } from "../../../hoc.js";
 import { useFetch, usePageTitle } from "../../../hooks.js";
 import logoDefiSaver from "../../../images/defisaver.svg";
 import { shorten } from "../../../utils/address.js";
 import { smartEtherscanUrl } from "../../../utils/url.js";
 import WalletActivityTable from "./components/WalletActivityTable.js";
+import WalletRawActivityTable from "./components/WalletRawActivityTable.js";
 import WalletInfo from "./components/WalletInfo.js";
 import WalletPositionsCard from "./components/WalletPositionsCard.js";
 import styles from "./Wallet.module.scss";
+import CurrencySwitch from "../../../components/CurrencySwitch/CurrencySwitch.js";
 
 function Wallet(props) {
   const { address } = useParams();
   const location = useLocation();
   usePageTitle(shorten(address));
 
+  const [isTokenCurrency, setIsTokenCurrency] = useState(false);
```

Comment:
Rename this variable to express the thing it actually does pls. Like `showRawEvents` or something similar

Reply:
Changed var names, function name and removed block and user address, since user address is the same as wallet address and block number is not relevant and can be accessed following tx_hash